Sloth.Core/Migrations/SlothDbContextModelSnapshot.cs

```diff
@@ -669,14 +670,16 @@ protected override void BuildModel(ModelBuilder modelBuilder)
                     b.Property<int>("Direction")
                         .HasColumnType("int");
 
+                    b.Property<string>("FinancialSegmentString")
+                        .HasColumnType("nvarchar(max)");
```

Comment:
@srkirkland Max? Maybe just 250 or so....

Reply:
ha i thought you'd say that.  and you are right too.  The largest it can be in GL string == 69 chars.  So I'll go 128 and call it good